Gruntfile.js

```diff
@@ -13,6 +16,51 @@ grunt.initConfig({
 	},
 	jscs: {
 		all: [ "*.js" ]
+	},
+	sass: {
+		dist: {
+			options: {
+				sourceMap: true,
+				outputStyle: "compressed"
+			},
+			files: [ {
+				expand: true,
+				cwd: "scss",
+				src: [ "*.scss" ],
+				dest: "",
+				ext: ".css"
+			} ]
+		}
+	},
+
+	autoprefixer: {
+		dist: {
+			options: {
+				map: true,
+
+				// Target-specific options go here.
+				// browser-specific info: https://github.com/ai/autoprefixer#browsers
```

Comment:
There comments seem more like documentation for autoprefixer then comments about what we are actually doing here.